ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -38,12 +38,12 @@ promotion:
 releases:
   initial:
     integration:
-      name: "4.10"
+      name: "4.11"
```

Comment:
does the bundle pullspec (line 33 above) need a similar replacement?

Reply:
I think so